app/services/export/continuing_activities.rb

```diff
@@ -23,6 +26,23 @@ def rows
       partner_organisation_name = activity.organisation.name
       activity_title = activity.title
       roda_identifier = activity.roda_identifier
+      fund = activity.associated_fund.roda_identifier
+      parent_level_b_roda_id = case activity.level
```

Comment:
It could, but given this is temporary code, and the most important part (the migration itself) hasn't even been started, I'm less inclined to make it perfect.